python/cudf/cudf/indexing.py

```diff
@@ -180,7 +180,7 @@ def _getitem_tuple_arg(self, arg):
             if isinstance(arg[0], slice):
                 start = arg[0].start
                 if start is None:
-                    start = 0
+                    start = self._df.self._df.index[0]
```

Comment:
`self._df.self._df`

Reply:
No idea how this got in there, will resolve...